### ISSUE TITLE
Failed to install by source code

### ISSUE THREAD

**Issue by Spacebody** [sakaiproject/sakai#3514]
Failed to install by source code because the file in googecode was not found any more.

**Comment by master-bob** [comment]
Can you please be more specific?

Also GitHub Issues are to only be used for problems related to the GradebookNG tool. All other issues are to be on https://jira.sakaiproject.org. Before creating a ticket, I suggest you make a post to the sakai-dev list, more info [here](https://confluence.sakaiproject.org/display/PMC/Sakai+email+lists).

**Comment by Spacebody** [comment]
I downloaded the source code in my server and followed the instructions to install by using "mvn install". But during the process of installation, it seemed  the script will retrieve the code from googlecode website, but then it came to be a problem that the code cannot be downloaded successfully and the script just tried again and again. I try to use the binary files to install, also met the same problem. I copied the link from the terminal and searched in the browser but just got 404 Not Found.
